### ISSUE TITLE
Froce font size(width and height) for iconfont

### ISSUE THREAD

**Issue by chenyinchang** [lvgl/lv_font_conv#90]
I have a problem about iconfont in a long time.
I use "lv_font_conv "  which convert a  “my_iconfont.c” file. And then try to modify ".c" file which have "offset x" and "offset y" for aligning icons to same column. But I try my best，you can see result below here ：
![mmexport1687156126177](https://github.com/lvgl/lv_font_conv/assets/33446056/b5bf21e0-518e-42f3-9c35-81a9a013b708)

So, I want to know how froce font size instead of [compact format](https://github.com/lvgl/lv_font_conv/blob/master/doc/font_spec.md)

Thanks

**Comment by kisvegabor** [comment]
Is the WiFi icon + text a single label in the example?

**Comment by chenyinchang** [comment]
> Is the WiFi icon + text a single label in the example?

no, split two objects (img + label) ， I use example : [Complex Menu](https://docs.lvgl.io/8.3/widgets/extra/menu.html#complex-menu)

**Comment by chenyinchang** [comment]
> > Is the WiFi icon + text a single label in the example?
> 
> no, split two objects (img + label) ， I use example : [Complex Menu](https://docs.lvgl.io/8.3/widgets/extra/menu.html#complex-menu). And I just change icon on my selection from [FontAwesome symbol font](https://lvgl.io/assets/others/FontAwesome5-Solid+Brands+Regular.woff) and set as 16px + 4bpp to convert as ".c" files.

Event the [Complex Menu](https://docs.lvgl.io/8.3/widgets/extra/menu.html#complex-menu) example icon look really good, But I change icon to something thin icon for example "LV_SYMBOL_BLUETOOTH" which seems offset x :
![20230619233311](https://github.com/lvgl/lv_font_conv/assets/33446056/57490bf3-406b-456b-bbf5-eae792d366fd)

**Comment by chenyinchang** [comment]
> > Is the WiFi icon + text a single label in the example?
> 
> no, split two objects (img + label) ， I use example : [Complex Menu](https://docs.lvgl.io/8.3/widgets/extra/menu.html#complex-menu). And I just change icon on my selection from [FontAwesome symbol font](https://lvgl.io/assets/others/FontAwesome5-Solid+Brands+Regular.woff) and set as 16px + 4bpp to convert as ".c" files.

As you can see, I change the partly code:
from img obj to lable obj which can adj padding for every icon
```
    if (icon) {
        //img = lv_img_create(obj);
        img = lv_label_create(obj);
        //lv_img_set_src(img, icon);
        lv_label_set_text(img, icon);
        lv_obj_set_style_text_font(img, &ui_font_FontAwesome5, LV_PART_MAIN | LV_STATE_DEFAULT);

        if (icon == "\xEF\x83\xAB")
        {
            lv_obj_set_style_pad_left(img, 3, LV_PART_MAIN | LV_STATE_DEFAULT);
        }
        if (icon == LV_SYMBOL_WIFI)
        {
            lv_obj_set_style_pad_left(img, -1, LV_PART_MAIN | LV_STATE_DEFAULT);
        }
    }
```
and I success **(but even though I think a group of iconfonts which in a 'n*n' piexl box center by user setting is really useful. That means I don't have to adjust every single icon padding one by one !)**
![success](https://github.com/lvgl/lv_font_conv/assets/33446056/f9f32e29-aa8d-4e2c-b17d-c6bce0ae2f7b)

**Comment by kisvegabor** [comment]
I think it should be solved in LVGL and not in the font converter. For v9 we will have more "image sizing options" which apply if the widget size of not the same as the mage source size. LVGL will be able to center, tile or zoom the image according to the widget size. In your case setting e.g. `lv_obj_set_width(img, 50)` and using a center aligned image source would solve the issue, but this feature is not implemented yet.

Until that I recommend this workaround in `create_text`:
```c
    if(icon) {
        lv_obj_t * wrapper = lv_obj_create(obj);
        lv_obj_remove_style_all(wrapper);
        lv_obj_set_size(wrapper, 50, LV_SIZE_CONTENT);
        img = lv_img_create(wrapper);
        lv_img_set_src(img, icon);
        lv_obj_center(img);
    }
```
